snuba/datasets/cdc/cdcprocessors.py

```diff
@@ -1,12 +1,37 @@
+from abc import ABC, abstractclassmethod
+from typing import Any, Mapping, Type
+
 from snuba.processor import MessageProcessor
 
 KAFKA_ONLY_PARTITION = 0  # CDC only works with single partition topics. So partition must be 0
 
 
+class CDCMessageRow(ABC):
```

Comment:
Casing is inconsistent between this and `CdcProcessor` below.

I don't have a specific suggestion as to which should be used (I think `CDCProcessor` looks a bit better but I could also see the argument that `CdcProcessor` is a bit less ambiguous) but we should settle on one style over another when dealing with acronyms in class names.

Reply:
will change to CdcMessageRow